src/components/radio-group/bl-radio-group.css

```diff
@@ -0,0 +1,21 @@
+:host {
+  display: flex;
+  flex-direction: row;
+}
+
+fieldset {
+  border: none;
+  padding: 0;
+}
+
+legend {
+  font: var(--bl-font-title-3-medium);
+  color: var(--bl-color-content-primary);
+}
+
+.options {
+  display: flex;
+  flex-flow: var(--bl-radio-direction, column) wrap;
+  gap: 16px;
```

Comment:
A variable set here ?

Reply:
What do you mean? User can set `--bl-radio-direction` but it's `column` by default.

Reply:
Sorry, I meant a variable can be set for the gap:16px

Reply:
I mean, it could but in design it was 16px and couldn't be sure if we would need an option for that. I think we can add it easily in the future if we see a need.